include/capstone/platform.h

```diff
@@ -62,6 +62,11 @@ typedef unsigned long long uint64_t;
 #define UINT64_MAX       0xffffffffffffffffui64
 #endif  // defined(_MSC_VER) && (_MSC_VER <= 1700 || defined(_KERNEL_MODE))
 
+#ifdef CAPSTONE_HAS_OSXKERNEL
```

Comment:
Why? It should be like it is. Kernel.framework provides stdint.h but does not provide inttypes.h.